specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -177,6 +232,61 @@
         }
       }
     },
+    "/managedidentitymoderestore": {
+      "put": {
+        "tags": [
+          "ManagedIdentityModeFullRestore"
+        ],
+        "operationId": "ManagedIdentityModeFullRestore",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
```

Comment:
Same comments here for LROs. Please following the [guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#obtaining-status-and-results-of-long-running-operations) for data-plane APIs.